Adyen/UI/Form/Items/Address/AddressStyle.swift

```diff
@@ -26,8 +24,30 @@ public struct AddressStyle: FormValueItemStyle {
     }
 
     /// The background color of the view.
-    public var backgroundColor: UIColor = .clear
+    public var backgroundColor: UIColor
 
     /// The color of form view item's separator line.
     public var separatorColor: UIColor? { textField.separatorColor }
+    
+    /// Initializes the form address item configuration.
+    /// - Parameters:
+    ///   - title: The section header style.
+    ///   - textField: The text field style.
+    ///   - tintColor: The tint color of the view.
+    ///   - backgroundColor: The background color of the view.
+    public init(
```

Comment:
This init looks a bit messy :), why change the prev one?
Or maybe we can add an empty init and a regular init and keep the default values as before?